src/services/azureFunctionsService.ts

```diff
@@ -68,7 +68,25 @@ export class AzureFunctionsService implements mssql.IAzureFunctionsService {
 		}
 		let projectFile = await azureFunctionUtils.getAzureFunctionProject();
 		if (!projectFile) {
-			vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened);
+			let projectCreate = await vscode.window.showErrorMessage(LocalizedConstants.azureFunctionsProjectMustBeOpened, constants.learnMore, constants.createProject);
+			if (projectCreate === constants.learnMore) {
+				vscode.commands.executeCommand('vscode.open', vscode.Uri.parse(constants.sqlBindingsDoc));
+			} else if (projectCreate === constants.createProject) {
+				// ask what folder the project should be created in
+				const options: vscode.OpenDialogOptions = {
+					title: constants.selectFolder,
+					canSelectMany: false,
+					openLabel: 'Select',
+					canSelectFiles: false,
+					canSelectFolders: true
+				};
+				// create the project based on the folder selected
+				await vscode.window.showOpenDialog(options).then(fileUri => {
```

Comment:
If you're already awaiting this here then there isn't any reason to use .then - just put that code after the await.